signature/mechanism_gpgme.go

```diff
@@ -1,5 +1,5 @@
-//go:build !containers_image_openpgp
-// +build !containers_image_openpgp
+//go:build !containers_image_openpgp && !containers_image_sequoia
+// +build !containers_image_openpgp,!containers_image_sequoia
```

Comment:
Should we default to sequoia or not?

---

Eventually the build tag, whichever tag it ends up being, should be documented in `README.md`.